cromulent/vocab.py

```diff
@@ -79,6 +79,7 @@ def register_instance(name, data):
 	"FoliationStatement": {"parent": LinguisticObject, "id":"300200662", "label": "Foliation Statement", "brief": True},
 	"Abstract": {"parent": LinguisticObject, "id":"300026032", "label": "Abstract", "brief": True},
 	"Note": {"parent": LinguisticObject, "id":"300027200", "label": "Note", "brief": True},
+	"Annotation": {"parent": LinguisticObject, "id":"300026100", "label":"Annotation"},
```

Comment:
OK for this to sneak into the PR for serialization?

Reply:
We should discuss how better to manage this sort of thing. It's more of a data file than actual code.